date

```diff
@@ -0,0 +1 @@
+ptk 18 kwi 16:07:15 2025 CEST
```

Comment:
What is propose of this file ?

Reply:
It's a temporary file from a script triggering new build (for external changes). 

Added a reminder in the description to remove along the dependency file!